### DISCUSSION TITLE
How to constrain to specific schemas?

### DISCUSSION THREAD

**Discussion by thasmo** [ariga/atlas#2254] [Q&A] [answered]
Is there a way to configure Atlas to work on specific schemas only?

I'm using Atlas with supabase and would like it to work on the `public` (postgresql) schema only.

**Comment by a8m** [comment, marked as answer]
Add the `search_path` query string to the URL. For example: `postgres://localhost:5432/database?search_path=public`

See: https://atlasgo.io/concepts/url

**Reply by thasmo** [reply]
Thanks! Thats seems to be another way to configure it; I just found out about the [`env.schemas`](https://atlasgo.io/atlas-schema/projects#environments) option in the [`atlas.hcl`](https://atlasgo.io/atlas-schema/projects) project configuration file, which helps to set this via configuration, I guess.
